shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -638,6 +640,8 @@ impl CassandraSinkCluster {
                         listen_address_alias = alias;
                     } else if column.name == Identifier::Unquoted("host_id".to_string()) {
                         host_id_alias = alias;
+                    } else if column.name == Identifier::Unquoted("rpc_port".to_string()) {
```

Comment:
I dont believe such a field exists:
![image](https://user-images.githubusercontent.com/5120858/189047489-0c74c4ff-3a08-4545-a7a8-648f3acf938d.png)

Reply:
Oops got mixed up with `system.local` clearly.

Reply:
Oh wait I was correct, the function that change is in is `rewrite_table_local`.

Reply:
Oh true, I got the tables mixed up.
rpc_address/rpc_port is actually the fields describing where the cql protocol connection goes. :+1: